recipes/libelfin/all/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 3.1)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+project(${CONAN_PACKAGE_NAME} VERSION ${CONAN_PACKAGE_VERSION})
+conan_basic_setup()
+
+file(GLOB_RECURSE elf_sources source_subfolder/elf/*.cc)
+set(elf_headers
+    source_subfolder/elf/common.hh
+    source_subfolder/elf/data.hh
+    source_subfolder/elf/elf++.hh)
+file(GLOB_RECURSE dwarf_sources source_subfolder/dwarf/*.cc)
+set(dwarf_headers
+    source_subfolder/dwarf/data.hh
+    source_subfolder/dwarf/dwarf++.hh
+    source_subfolder/dwarf/small_vector.hh)
+
+add_custom_command(
+    OUTPUT source_subfolder/elf/to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo '// Automatically generated' > to_string.cc
```

Comment:
I think you are mixing hard tabs with spaces.

Reply:
Fixed, thanks.